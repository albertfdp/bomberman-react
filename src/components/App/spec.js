import React from 'react'
import expect from 'test/expect'

import { App } from '.'

const noop = () => {}

describe('App', () => {
  it('renders', () => {
    expect(<App dispatch={noop} />, 'to match snapshot')
  })
})
