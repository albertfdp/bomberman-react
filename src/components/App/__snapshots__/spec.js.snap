exports[`App renders 1`] = `
<div
  className="container" />
`;
